samples/mpu/mem_domain_apis_test/README.rst

```diff
@@ -0,0 +1,80 @@
+.. _mem_domain_apis_test:
+
+Memory Domain APIs Test
+#######################
+
+Overview
+********
+
+This is a simple application that demonstrates the usage of Memory Domain APIs.
+Two memory domains (app0_domain and app1_domain) and three application threads
+(app thread 0, 1, 2) are created by the main thread.
+Each application thread will add itself into a memory domain while it starts.
+These threads will yield their execution alternately and we can observe the
+changes of domain partition regions during context switch via the serial console.
+
+Building and Running
+********************
+
+This project can be built and executed as follows:
+
+.. code-block:: console
+
+   $ cd samples/mpu/mem_domain_apis_test
+   $ make BOARD=stm32f4_disco
```

Comment:
should be 'make BOARD=stm32f4_disco'. I just want to show how to build it.
and there is a statement 'Flash the generated zephyr.bin on the board' after it
to remind developer to flash new image file.